src/Plugin/EntityReferenceSelection/OgSelection.php

```diff
@@ -37,11 +37,15 @@ class OgSelection extends DefaultSelection {
    * @return DefaultSelection
    */
   public function getSelectionHandler() {
-    $field = $this->getConfiguration('field');
-
-    if ($field) {
-      return \Drupal::service('plugin.manager.entity_reference_selection')->getSelectionHandler($field);
```

Comment:
```php
$field = $this->getConfiguration('field');
```

Will never be available when being used from a form autocomplete element, but the target type and handler settings are, and from that we can get a base selection handler.